content/docs/typechecking-with-proptypes.md

```diff
@@ -196,3 +196,47 @@ class Greeting extends React.Component {
 ```
 
 The `defaultProps` will be used to ensure that `this.props.name` will have a value if it was not specified by the parent component. The `propTypes` typechecking happens after `defaultProps` are resolved, so typechecking will also apply to the `defaultProps`.
+
+### Functional Components
+
+If you are using Functional Components in your regular development, you may want to make some small changes to allow PropTypes to be proper applied.
```

Comment:
```suggestion
If you are using function components in your regular development, you may want to make some small changes to allow PropTypes to be proper applied.
```

"function components" isn't treated as a proper noun in the docs as far as I can tell.